src/components/CourseCard/BaseCourseCard.jsx

```diff
@@ -93,32 +95,57 @@ class BaseCourseCard extends Component {
           </div>
           {children && children}
           {hasEmailSettings && (
-            <div className="row no-gutters">
-              <button
-                className="btn btn-link p-0"
-                onClick={() => this.setModalState({
-                  key: 'email_settings',
-                  options: {
-                    title: `Email Settings for '${title}'`,
-                    submitting: false,
-                    data: {
-                      course: linkToCourse,
-                    },
-                  },
-                  })}
-              >
-                <FontAwesomeIcon className="mr-2" icon={['fas', 'cog']} />
-                Email settings
-              </button>
-              {
-                modals.email_settings &&
-                <EmailSettingsModal
-                  {...modals.email_settings}
-                  onEmailSettingsChange={this.handleEmailSettingsChange}
-                  onClose={this.resetModals}
-                />
+            <>
+              {hasNewEmailSettings &&
+                <div className="row no-gutters">
+                  <div className="col-xs-12 col-sm-10 col-md-8">
+                    <StatusAlert
+                      alertType="success"
+                      dialog={
+                        <>
+                          <FontAwesomeIcon className="mr-2" icon={['fas', 'check-circle']} />
+                          Your email settings have been saved.
+                        </>
+                      }
+                      onClose={() => {
+                        this.setState({
+                          hasNewEmailSettings: false,
+                        });
+                      }}
+                      open
+                    />
+                  </div>
+                </div>
               }
-            </div>
+              <div className="row no-gutters">
+                <div className="col">
+                  <button
+                    className="btn btn-link p-0"
+                    onClick={() => {
+                      this.setModalState({
+                        key: 'emailSettings',
+                        options: {
+                          title,
+                          hasEmailsEnabled,
```

Comment:
Pass `hasEmailsEnabled` to `EmailSettingsModal` so that the initial `checked` state of the email settings checkbox reflects the actual value provided by the API.